modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -220,6 +223,17 @@ const questionnaireStyle = theme => ({
         opacity: 1,
         zIndex: "1010"
     },
+    formFooter: {
+        position: "sticky",
+        bottom: 0,
```

Comment:
I don't like how the form can be seen in the background under the sticky progress bar.
![captured (7)](https://user-images.githubusercontent.com/651980/96658126-c9721200-1311-11eb-9580-315e984aaf5a.gif)
What about:
```suggestion
        bottom: theme.spacing(-2),
```
to compensate for the bottom padding?